sources/packages/backend/apps/api/src/route-controllers/overaward/_tests_/e2e/overaward.institution.controller.getOverawardBalance.e2e-spec.ts

```diff
@@ -0,0 +1,64 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
```

Comment:
The file has to be renamed with institution in plural .

overaward.**institutions**.controller.getOverawardsByStudent.e2e-spec.ts